File: python/questao5.py
def gerar_email(nome_completo):
    nome, sobrenome = nome_completo.split()[-1], nome_completo.split()[0]
    print(f"Sr. {sobrenome}, seu email é {nome}@lp.com.br")

nome_completo = input("Digite seu nome completo: ")
gerar_email(nome_completo)

File: c/questao5.c
#include <stdio.h>
#include <string.h>

void gerarEmail(char *nomeCompleto) {
    char *sobrenome = strrchr(nomeCompleto, ' ');
    if (sobrenome != NULL) {
        *sobrenome = '\0'; 
        printf("Sr. %s, seu email é %s@lp.com.br\n", nomeCompleto, nomeCompleto);
    } else {
        printf("Nome incompleto.\n");
    }
}

int main() {
    char nomeCompleto[100];
    printf("Digite seu nome completo: ");
    fgets(nomeCompleto, sizeof(nomeCompleto), stdin);
    nomeCompleto[strcspn(nomeCompleto, "\n")] = '\0'; 
    gerarEmail(nomeCompleto);
    return 0;
}
